### DISCUSSION TITLE
How to get in touch

### DISCUSSION THREAD

**Discussion by yayoi-chi** [jjon-saxton/yaya-world#9] [General]
If you need help in one of the MMOs I play, or want to get in-touch with me, you can actually just add a comment to this discussion. Or you can get a hold of me by [email](mailto:yayoi_chi@icloud.com?subject=Help%20Me), or through [matrix](https://matrix.to/#/@yayoichi:matrix.org) or [Mastodon](https://mastodon.world/@yaya).

**Comment by jjon-saxton** [comment]
Looks like you've been busy ;)

**Reply by yayoi-chi** [reply]
Just a bit 😝
